lib/code-ribbon-ribbon-container.js

```diff
@@ -39,6 +39,11 @@ class CodeRibbonRibbonContainer extends PaneAxis {
   initialize() {
     this.cr_update();
     this.getElement().classList.add("cr-ribbon-container");
+    
+    // TODO: Why would this not fire??
```

Comment:
@tkeyes25 because a RibbonContainer does not scroll, it is a fixed size element.

A CR-Primary-Container does scroll however... (CR Manager) since it contains the oversized RibbonContainer